hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -1024,6 +1026,7 @@ void generateSnapshotDiffReport(final String jobKey,
       throw new RuntimeException(exception);
     } catch (Exception exception) {
       updateJobStatusToFailed(jobKey, exception.getMessage());
+      ozoneManager.getMetrics().incNumSnapshotDiffJobFails();
```

Comment:
Do you want to move it inside `updateJobStatusToFailed` so that we can avoid duplication?